server/services/cohorts.js

```diff
@@ -60,7 +60,7 @@ const getCohortParticipants = async (cohortId) =>
       },
     })
     .find({
-      'cohortParticipantsJoin.cohort_id': cohortId,
+      cohort_id: cohortId,
       'participantStatusJoin.current': true,
       'participantStatusJoin.status': ['hired', 'archived'],
```

Comment:
I'm not totally sure if this is the problem, but you may be able to rework the query to take advantage of the count function, which you can use to count unique ids (but is incompatible with .find()). The conditions in the `find` can usually be moved into the joins.